site/ja/clients/WebSocket.md

```diff
@@ -275,20 +275,19 @@ var message = {
 };
 
 ws.on('open', function() {
-  // 1つのメッセージを送信します
-  ws.send(JSON.stringify(message));
+  // Send one message
```

Comment:
this is a Japanese documentation page. you don't need to change it :)

Reply:
> this is a Japanese documentation page. you don't need to change it :)

ok